src/include/mallocMC/reservePoolPolicies/SimpleMalloc.hpp

```diff
@@ -1,11 +1,5 @@
 /*
   mallocMC: Memory Allocator for Many Core Architectures.
-  https://www.hzdr.de/crp
-
-  Copyright (C) 2014 Institute of Radiation Physics,
-                     Helmholtz-Zentrum Dresden - Rossendorf
-
-  Author(s):  Carlchristian Eckert - c.eckert ( at ) hzdr.de
```

Comment:
I guess you want something here from me? Like copyright CASUS and CERN and my name.

Reply:
yes would be nice

Reply:
done